index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Document</title>
+  <link rel="stylesheet" href="./styles.css">
+</head>
+<body>
+  <nav>
+    <div class="nav-bar">
+      <h1 class="logo">Kajal</h1>
+      <img src="./img/hambug.png" alt="hamburger" class="hambug">
```

Comment:
- [ ] it would be better if you wrap your hamburger inside a `<button>` tag cause actually its a button always try to use the right semantic tags that will improve the readability and UX